runtime/kusama/src/weights/runtime_parachains_paras_inherent.rs

```diff
@@ -146,10 +145,10 @@ impl<T: frame_system::Config> runtime_parachains::paras_inherent::WeightInfo for
 	// Storage: Hrmp HrmpWatermarks (r:0 w:1)
 	// Storage: Paras Heads (r:0 w:1)
 	fn enter_backed_candidates_variable(v: u32, ) -> Weight {
-		(319_740_000 as Weight)
-			// Standard Error: 26_000
-			.saturating_add((49_051_000 as Weight).saturating_mul(v as Weight))
-			.saturating_add(T::DbWeight::get().reads(29 as Weight))
+		(1_134_397_000 as Weight)
```

Comment:
3x increase to the static weight here is quite significant

Reply:
I don't quite understand where that comes from